src/main/java/sirius/biz/jobs/interactive/ReportJobFactory.java

```diff
@@ -25,6 +27,9 @@
  */
 public abstract class ReportJobFactory extends InteractiveJobFactory {
 
+    @Part
+    protected Cells cells;
```

Comment:
why?

Reply:
required by all subclasses to populare the report - documentation has been expanded.